### PR TITLE
Refactor CSS into separate files using BEM naming

### PR DESCRIPTION
**Pull request by hnryjmes**
HTML and CSS changes including:
- CSS now in separate files according to specificity
- CSS selectors follow BEM (Block Element Modifier) convention

### CODE REVIEW

File: app/enquiries/templates/base.html [hunk 1]
```diff
@@ -18,7 +18,11 @@
   <link rel="apple-touch-icon" href="{% static 'assets/images/govuk-apple-touch-icon.png' %}">
 
   <!--[if !IE 8]><!-->
-  <link href="{% static 'css/application.css' %}" rel="stylesheet">
+  <link href="{% static 'css/enquiry_delete.css' %}" rel="stylesheet">
```
Comment (ian-leggett) [inline app/enquiries/templates/base.html]: You shouldn't need separate CSS files here, I would bundle the dist version as `application.css`

File: frontend/sass/enquiries.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+.govuk-header__content {
+    padding-left: 0px;
+}
+
+.govuk-label {
+    font-weight: bold;
+}
+
+ol {
```
Comment (ian-leggett) [inline frontend/sass/enquiries.scss]: If this is a global rule then maybe this needs to go into a `settings.scss` look at https://github.com/alphagov/govuk-frontend/tree/master/src/govuk/settings

File: frontend/sass/enquiries.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+.govuk-header__logo, .govuk-header__content {
+    display: inline-block;
+}
+
+.govuk-header__container {
+    border-bottom: 10px solid white;
+}
+
+.govuk-header__link--service-name {
+    font-size: 30px;
```
Comment (ian-leggett) [inline frontend/sass/enquiries.scss]: Is this an official Gov font size?

File: frontend/sass/enquiry_delete.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
+$govuk-assets-path: '/static/assets/';
+
+@import "node_modules/govuk-frontend/govuk/all";
```
Comment (ian-leggett) [inline frontend/sass/enquiry_delete.scss]: We don't need to import this here, this should be in the root SCSS file such as `application.scss`

File: frontend/sass/enquiry_detail.scss [hunk 5]
```diff
@@ -0,0 +1,16 @@
+$govuk-assets-path: '/static/assets/';
+
+@import "node_modules/govuk-frontend/govuk/all";
```
Comment (ian-leggett) [inline frontend/sass/enquiry_detail.scss]: See above comment

File: frontend/sass/enquiry_edit.scss [hunk 6]
```diff
@@ -0,0 +1,4 @@
+$govuk-assets-path: '/static/assets/';
+
+@import "node_modules/govuk-frontend/govuk/all";
```
Comment (ian-leggett) [inline frontend/sass/enquiry_edit.scss]: See above comment

File: frontend/sass/enquiry_list.scss [hunk 7]
```diff
@@ -0,0 +1,95 @@
+$govuk-assets-path: '/static/assets/';
+
+@import "node_modules/govuk-frontend/govuk/all";
```
Comment (ian-leggett) [inline frontend/sass/enquiry_list.scss]: See above comment

**Comment by saruniitr** [comment]
Thanks for your reviews, received two approvals, merging it

**Pull request merged**